src/components/atoms/TagNavigatorListItem.tsx

```diff
@@ -52,36 +72,64 @@ const TagNavigatorListItem = ({
   noteId,
   currentTagName,
   removeTagByName,
+  updateTagColorByName,
 }: TagNavigatorListItemProps) => {
   const { t } = useTranslation()
   const { push } = useRouter()
   const { report } = useAnalytics()
 
+  const [colorPickerModal, showColorPickerModal] = useState(false)
+
+  const openTagContextMenu: React.MouseEventHandler = useCallback(
+    (_) => {
```

Comment:
If you don't need to access the event parameter, you don't need to provide the handler type

Reply:
Removed!